src/core/authentication/authentication.service.spec.ts

```
@@ -16,26 +105,515 @@ describe('AuthService', () => {
 
     service = module.get(AuthenticationService);
+    agentInfoCacheService = module.get(AgentInfoCacheService);
+    agentInfoService = module.get(AgentInfoService);
+    kratosService = module.get(KratosService);
+    agentService = module.get(AgentService);
+    configService = module.get(ConfigService);
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
   });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)